src/screens/Home/index.tsx

```diff
@@ -67,7 +66,7 @@ const Home = () => {
   );
 
   return (
-    <VStack>
+    <VStack px={8}>
```

Comment:
avoid magical numbers. Use constants from src/styles/theme.ts instead

Reply:
I created a file to store sizes constants